docs/dataproc.md

```diff
@@ -0,0 +1,99 @@
+## Integration with DataProc / PySpark
+
+Bigflow provides integration with Pyspark running on Dataproc.
+It allows to adapt any python callable/function with PySpark code into `dataflow` Job.
```

Comment:
Maybe something more generic, like - "It allows you to easily build, run, configure, and schedule a Dataproc job."

Reply:
Agree, changed text.